chapters/th/_toctree.yml

```diff
@@ -0,0 +1,21 @@
+- title: 0. ติดตั้งโปรแกรม
+  sections:
+  - local: chapter0/1
+    title: บทนำ
+
+- title: 1. โมเดล Transformer
+  sections:
+  - local: chapter1/1
+    title: บทนำ
+
+- title: 2. การใช้ 🤗 Transformers
+  sections:
+  - local: chapter2/1
+    title: บทนำ
+  - local: chapter2/2
```

Comment:
We should only include the chapters / sections in the ToC for content that has already been translated. So in this case you can remove `chapter2/2` and Chapter 3 :)

Reply:
Hi @ckingkan would you still like to work on this last change or would you prefer that I take care of it (happy to do so!)

Reply:
Hi @lewtun, sorry for my late reply. I have fixed the conflicts